sunkit_image/new_coalignment_api_dev/coalignment_interface.py

```diff
@@ -0,0 +1,192 @@
+import warnings
+
+import astropy.units as u
+import numpy as np
+import sunpy.map
+from skimage.feature import match_template
+from sunpy.util.exceptions import SunpyUserWarning
+
+
```

Comment:
Can you add the __all__ for the public functions please?

Reply:
Surely